src/sentry/runner/commands/presenters/slackpresenter.py

```diff
@@ -136,12 +117,19 @@ def invalid_type(
         expected_type: type,
     ) -> None:
         self.invalid_type_options.append((key, got_type, expected_type))
+        self.options_changed = True
 
     def _send_to_webhook(self, json_data: dict) -> None:
         if settings.OPTIONS_AUTOMATOR_SLACK_WEBHOOK_URL:
             headers = {"Content-Type": "application/json"}
-            requests.post(
-                settings.OPTIONS_AUTOMATOR_SLACK_WEBHOOK_URL,
-                data=json.dumps(json_data),
-                headers=headers,
-            )
+            try:
+                response = requests.post(
+                    settings.OPTIONS_AUTOMATOR_SLACK_WEBHOOK_URL,
+                    data=json.dumps(json_data),
+                    headers=headers,
+                )
+                response.raise_for_status()
+            except HTTPError as http_err:
+                raise ConnectionError(f"HTTP error occurred: {http_err}") from None
+            except RequestException as req_err:
+                raise ConnectionError(f"Network-related error occurred: {req_err}") from None
```

Comment:
Why `from None` ? You have an exception to propagate. If you raise `from None` the stack trace is going to be lost, while if you provide a proper `from` clause passing the source exception, THe error in the logs and in Sentry will contain both, thus allow you to troubleshoot the issue.